plugins/woocommerce-admin/client/payments-welcome/strings.tsx

```diff
@@ -378,4 +378,59 @@ export default {
 
 		getInTouch: __( 'Get in touch', 'woocommerce' ),
 	},
+	apms: {
+		addMoreWaysToPay: __(
+			'Add more ways for buyers to pay',
+			'woocommerce'
+		),
+		seeMore: __( 'See more', 'woocommerce' ),
+		paypal: {
+			title: __( 'PayPal Payments', 'woocommerce' ),
+			description: __(
+				'Enable PayPal Payments alongside WooCommerce Payments. Give your customers another way to pay safely and conveniently via PayPal, PayLater, and Venmo.',
+				'woocommerce'
+			),
+		},
+		amazonpay: {
+			title: __( 'Amazon Pay', 'woocommerce' ),
+			description: __(
+				'Enable Amazon Pay alongside WooCommerce Payments and give buyers the ability to pay via Amazon Pay. Transactions take place via Amazon embedded widgets, so the buyer never leaves your site.',
+				'woocommerce'
+			),
+		},
+		klarna: {
+			title: __( 'Klarna', 'woocommerce' ),
+			description: __(
+				'Enable Klarna alongside WooCommerce Payments. With Klarna Payments buyers can choose the payment installment option they want, Pay Now, Pay Later, or Slice It. No credit card numbers, no passwords, no worries.',
+				'woocommerce'
+			),
+		},
+		affirm: {
+			title: __( 'Affirm', 'woocommerce' ),
+			description: __(
+				'Enable Affirm alongside WooCommerce Payments and give buyers the ability to pick the payment option that works for them and their budget — from 4 interest-free payments every 2 weeks to monthly installments.',
+				'woocommerce'
+			),
+		},
+		installText: ( extensionsString: string ) => {
+			const extensionsPlural =
+				extensionsString.indexOf( ',' ) > -1
+					? 'extensions'
+					: 'extension';
+			return createInterpolateElement(
+				sprintf(
+					__(
+						'Installing <strong>WooCommerce Payments</strong> will automatically activate <strong>%s</strong> %s in your store.',
```

Comment:
Can you add a "translators" comment like this? Thanks!

https://github.com/woocommerce/woocommerce/blob/a0c13abd839665ca0acd02b5166df130b35c83ee/plugins/woocommerce-admin/client/homescreen/activity-panel/stock/card.js#L95

Reply:
Good catch, thank you! Will add that

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/34581/commits/579b1d63f1d8fdb76ca4449af6d23d873e438f09!